### PR TITLE
Add module lvm_info

### PR DESCRIPTION
**Pull request by danielino**
##### SUMMARY
This module reports lvm and disks configurations.

```
 ansible -i 192.168.56.10, all -m lvm_info -a "type=lv"
192.168.56.10 | SUCCESS => {
    "changed": false, 
    "failed": false, 
    "result": [
        {
            "attr": "-wi-ao----", 
            "lsize": "50,00g", 
            "lv": "root", 
            "vg": "centos"
        }, 
        ....
    ]
}

```
##### ISSUE TYPE
 - New Module Pull Request

##### COMPONENT NAME
modules/system/lvm_info.py

##### ANSIBLE VERSION

```
ansible 2.4.1.0
  config file = None
  configured module search path = [u'/Users/danielino/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/danielino/anaconda2/lib/python2.7/site-packages/ansible
  executable location = /Users/danielino/anaconda2/bin/ansible
  python version = 2.7.11 |Anaconda 2.4.1 (x86_64)| (default, Dec  6 2015, 18:57:58) [GCC 4.2.1 (Apple Inc. build 5577)]
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: lib/ansible/modules/system/lvm_info.py [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+    )
+
+    result = dict(
+        changed=False,
+        result=TypeSwitcher().switch(module.params['type'], module.params['name'])
+    )
+
+    module.exit_json(**result)
+
+main()
```
Comment (mattclay) [inline lib/ansible/modules/system/lvm_info.py]: Missing conditional on execution of `main`:

```python
if __name__ == '__main__':
    main()
```

**Comment by alikins** [comment]
Seems like this should be a facts module? ala https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/storage/zfs/zfs_facts.py or https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/storage/netapp/netapp_e_facts.py

**Comment by Akasurde** [comment]
@danielino Any news ?

**Comment by danielino** [comment]
yes, you're right. it's seems like a fact module. how can i submit as  a fact module? i need to close the issue and open a new one on the current ansible version?

**Comment by relrod** [comment]
This already exists in current facts, so closing.

**Pull request closed without merging**